src/components/pages/AdminDashboard/DevToolsNew.js

```diff
@@ -0,0 +1,91 @@
+import React from 'react';
+import { useHistory } from 'react-router-dom';
+
+import { Menu, Dropdown, Button } from 'antd';
+import { DownOutlined } from '@ant-design/icons';
+
+const DevToolsNew = props => {
+  const { push } = useHistory();
+
+  const SatHandler = () => {
+    push('/saturday');
+  };
+  const SunHandler = () => {
+    push('sunday');
+  };
+  const MonHandler = () => {
+    push('/monday');
+  };
+  const TuesHandler = () => {
+    push('tuesday');
+  };
+  const WedHandler = () => {
+    push('wednesday');
+  };
+  const ThursHandler = () => {
+    push('thursday');
+  };
+  const FriHandler = () => {
+    push('friday');
+  };
+
+  const dropdown = (
```

Comment:
You could also make this Menu dynamic and reusable instead of having to manually write in all of the item parts.  i.e. have it map through dayData.js and for each day have it render the relevant menu item.  for ex:
daysData.map(day => <Menu.Item><a>{dayName}</a></Menu.Item>)

You could pass in the url as a prop and have a clickhandler function in the component that pushes the history to the url passed in to props rather than pushing to a hardcoded in url.  This could really trim down the logic.  In that case, I might consider making the Menu its own component or making the MenuItem its own reusable component.  

Thoughts?

Reply:
This is also a great recommendation! I'll definitely implement the dynamic menu after I finish creating the reusable day component.